test/integration/functional_test.go

```diff
@@ -161,7 +161,7 @@ func validateNodeLabels(ctx context.Context, t *testing.T, profile string) {
 // check functionality of minikube after evaling docker-env
 func validateDockerEnv(ctx context.Context, t *testing.T, profile string) {
 	defer PostMortemLogs(t, profile)
-	mctx, cancel := context.WithTimeout(ctx, Seconds(30))
```

Comment:
let change it to 2 minutes.
since docker-env could invoke a api-restart if there is a flaky inner docker, better to wait for it.